sandbox/StatusExpandableSettingsItemPage.qml

```diff
@@ -0,0 +1,69 @@
+import QtQuick 2.14
+import QtQuick.Layouts 1.14
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+
+Column {
+    spacing: 12
+    width: 800
+    anchors.top:parent.top
+    leftPadding: 20
+    rightPadding: 20
+
+    StatusExpandableSettingsItem {
+        anchors.horizontalCenter: parent.horizontalCenter
+
+        expandable: false
+        icon.name: "seed-phrase"
+        primaryText: qsTr("Back up seed phrase")
+        secondaryText: qsTr("Back up your seed phrase now to secure this account ajhaDH SDHSAHDLSADBSA,DLISAHDLASD ADASDHASLDHALSDHAS DAS,DASJDGLIASGD")
+        button.text: qsTr("Back up seed phrase")
+    }
+
+    StatusExpandableSettingsItem {
+        anchors.horizontalCenter: parent.horizontalCenter
+
+        expandable: true
```

Comment:
Here's another idea, curious what you think:

We could potentially get rid off this property as well, if we rely on the heuristic that the component is always expandable when there's a `expandableComponent`

Wdyt?

Reply:
As discussed, lets keep this explicit for the user of this component